hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -6,7 +6,7 @@ group_eval_order:
   - kubevirt.cnao.rules
 
 tests:
-# CnaoDown positive tests
```

Comment:
Could we just remove the commit that this commit reverts?

Reply:
done